site/docs/v1/tech/themes/index.adoc

```diff
@@ -75,6 +75,31 @@ This template contains all of the main helper macros to define the `head`, `body
 +
 See <<Helpers>> section below for additional information.
 
+[field]#Account edit# [required]#Required# [since]#Since 1.26.0#::
+This page contains a form that enables authenticated users to update their profile.
++
+[uri]#/account/edit#
+
+[field]#Account index# [required]#Required# [since]#Since 1.26.0#::
+This is the self-service account landing page. An authenticated user may use this as a starting point for operations such as updating their profile or configuring multi-factor authentication.
++
+[uri]#/account#
+
+[field]#Account two-factor disable# [required]#Required# [since]#Since 1.26.0#::
+This page contains a form that accepts a verification code used to disable a multi-factor authentication method.
++
+[uri]#/account/two-factor/disable#
+
+[field]#Account two-factor enable# [required]#Required# [since]#Since 1.26.0#::
+This page contains a form that accepts a verification code used to enable a multi-factor authentication method.
++
+[uri]#/account/two-factor/enable#
```

Comment:
Is the recovery code page themeable?

Reply:
Recovery codes is part of `/account/two-factor/enable`.  Unfortunately its part of a conditional block so it doesn't display in the preview mode.

Reply:
might be worth noting that.

Reply:
How does this sound added to the description:

"Additionally, this page contains presentation of recovery codes when a user enables multi-factor authentication for the first time."

Reply:
👍 !